docs/bpmn-support.adoc

```diff
@@ -143,6 +143,7 @@ The event definition can be defined on the event or on the definitions.
 * message: https://github.com/jgraph/drawio/blob/0e19be6b42755790a749af30450c78c0d83be765/src/main/webapp/shapes/bpmn/mxBpmnShape2.js#L465[draw.io bpmn mxgraph stencil]
 * signal: https://thenounproject.com/term/triangle/2452089/[triangle] By https://thenounproject.com/imamdji99[Imam], ID from https://thenounproject.com 
 * timer: https://www.flaticon.com/free-icon/clock_223404[Timer Icon] "Icons made by https://www.flaticon.com/authors/kirill-kazachek[Kirill Kazachek] from https://www.flaticon.com"
+* compensation: https://thenounproject.com/vityavorobyev/icon/443401/[Rewind Button ] "Icons made by https://thenounproject.com/vityavorobyev/[Viktor Vorobyev] from https://thenounproject.com/"
```

Comment:
:question: can you move this on top? this is to sort the elements in alphabetic order

Reply:
Done. 
And for the question about the stroke size, I don't know, I tried with the two icons you proposed: https://github.com/process-analytics/bpmn-visualization-js/issues/355#issuecomment-698181743